src/features/swap/components/SwapOrder/index.tsx

```diff
@@ -105,7 +113,7 @@ export const SellOrder = ({ order }: { order: SwapOrderType }) => {
           <OrderId orderId={uid} href={explorerUrl} />
         </DataRow>,
         <DataRow key="Status" title="Status">
-          <StatusLabel status={status} />
+          <StatusLabel status={isPartiallyFilled ? 'partiallyFilled' : status} />
```

Comment:
nit: This could be extracted into a `getOrderStatus` helper together with the other helper functions so that the logic is in one place.

Reply:
It's a bit confusing and that's why I left like this.
The orderbook api only returns an expired status, there is no partiallyFilled. PartiallyFilled is a "meta" status only when expired and filled > 0. 
defining a function named `getOrderStatus` could imply that this is the order status, but it isn't. 
defining a function named `getOrderMetaStatus` could work, but I would have to explain what a "metaStatus" is, so I opted to have the more direct thing in the StatusLabel as this is basically the only place where we care about it.